content/development/flightRules/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Flight Rules
+description: Flight rules for Altinn Studio
+tags: ["flight","rules",  "altinn", "studio"]
+weight: 100
+---
+
+{{% notice info %}}
+
+**What are flight rules?**  
+Flight rules is originally a compendium of step-by-step manuals, what to do when X occurs and why.  
+Essentially, they are extremely detailed, scenario-specific standard operating procedures
+
+{{% /notice %}}
+
+### Loadbalancer is responding with HTTP Error 502 - bad gateway
+
+This occurs when the loadbalancer could not find any of the services in it's configuration.  
+Can be a source of these issues:
```

Comment:
source can be termed as "root cause"

Reply:
changed to `Can be triggerd by these issues:` instead, sounds better.